app/workers/tweet_worker.rb

```diff
@@ -0,0 +1,47 @@
+# Polls weather for subscribed locations
+class TweetWorker
```

Comment:
I like that you split out these methods into a separate object but its not really a worker because its not doing any polling. I'd be fine with this living in `app/clients` for now because its data is closely related to a client.